resources/core/charts/cluster-users/templates/clusterroles-k8s-io.yaml

```diff
@@ -0,0 +1,128 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: kyma-authorization-self
+  labels:
+    app: kyma
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    rbac.authorization.kyma-project.io/aggregate-to-kyma-authorization-self: "true"
+  annotations:
+    "helm.sh/hook-weight": "0"
+rules:
+- apiGroups:
+    - "authorization.k8s.io"
+  resources:
+    - "selfsubjectrulesreviews"
+    - "selfsubjectaccessreviews"
+  verbs:
+    - "create"
+
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: kyma-authorization-subject
+  labels:
+    app: kyma
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    rbac.authorization.kyma-project.io/aggregate-to-kyma-authorization-subject: "true"
+  annotations:
+    "helm.sh/hook-weight": "0"
+rules:
+- apiGroups:
+    - "authorization.k8s.io"
+  resources:
+    - "subjectaccessreviews"
+  verbs:
+    - "create"
+
+---
+#CRD management
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: kyma-crd
+  labels:
+    app: kyma
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    rbac.authorization.kyma-project.io/aggregate-to-kyma-crd: "true"
+  annotations:
+    "helm.sh/hook-weight": "0"
+rules:
+- apiGroups:
+    - "apiextensions.k8s.io"
+  resources:
+    - "customresourcedefinitions"
+  verbs:
+    - "create"
+    - "delete"
+    - "deletecollection"
+    - "get"
+    - "list"
+    - "patch"
+    - "update"
+    - "watch"
+
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: kyma-crd-list
```

Comment:
This might me more useable as `kyma-crd-view`, with `list, get` verbs. List alone seems lacking

Reply:
I don't want to change existing setup - this is only refactoring PR. Changes/fixes/improvements will be introduced in a separate PR.